keadm/cmd/keadm/app/cmd/reset.go

```diff
@@ -52,6 +53,68 @@ func newResetOptions() *common.ResetOptions {
 	return opts
 }
 
+func newKubeEdgeResetBeta() *cobra.Command {
+	IsEdgeNode := false
+	reset := newResetOptions()
+
+	var cmd = &cobra.Command{
+		Use:     "reset-beta",
+		Short:   "Teardowns KubeEdge (cloud(helm installed) & edge) component, is in beta trial use",
+		Long:    resetLongDescription,
+		Example: resetExample,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			whoRunning, err := util.RunningModuleV2(reset)
+			if err != nil {
+				return err
+			}
+			switch whoRunning {
+			case common.KubeEdgeEdgeRunning:
+				IsEdgeNode = true
+			case common.NoneRunning:
+				fmt.Println("None of KubeEdge components are running in this host, exit")
+				os.Exit(0)
+			}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if !reset.Force {
+				fmt.Println("[reset] WARNING: Changes made to this host by 'keadm init' or 'keadm join' will be reverted.")
+				fmt.Print("[reset] Are you sure you want to proceed? [y/N]: ")
+				s := bufio.NewScanner(os.Stdin)
+				s.Scan()
+				if err := s.Err(); err != nil {
+					return err
+				}
+				if strings.ToLower(s.Text()) != "y" {
+					return fmt.Errorf("aborted reset operation")
+				}
+			}
+			// 1. kill cloudcore/edgecore process.
+			// For edgecore, don't delete node from K8S
+			if err := TearDownKubeEdgeV2(IsEdgeNode, reset.Kubeconfig); err != nil {
```

Comment:
1 helm delete releaseName -n namespace
2 kubectl delete ns or etcd delete ns
3 In other cases, if there have workloads on the edges, edges must reset firstly. We can add docs for this case.

Reply:
1. https://github.com/kubeedge/kubeedge/tree/master/build/helm/cloudcore#uninstall
we should run `helm uninstall` cmd ?
2. the previous function `keadm reset` doesn't process evict the pods, only remove all the pods and database on the edge

Reply:
> we should run `helm uninstall` cmd ?

Because  init-beta will install as a chart, we had better do that.

> the previous function keadm reset doesn't process evict the pods, only remove all the pods and database on the edge

Does the previous function work on the cloud? If so, we need to implement it and add a parameter like compoent=cloud/edge to split the cloud reset and edge reset.